pkg/occlient/occlient.go

```diff
@@ -1936,16 +1938,18 @@ func (c *Client) GetClusterServiceClass(serviceName string) (*scv1beta1.ClusterS
 		FieldSelector: fields.OneTermEqualSelector("spec.externalName", serviceName).String(),
 	}
 	searchResults, err := c.serviceCatalogClient.ClusterServiceClasses().List(opts)
+
+	items := filterHiddenClasses(searchResults.Items)
```

Comment:
To be honest, I don't think this should be done. 
From what I understand, `hidden` is meant only to hide things when all services are listed, not when obtaining info in other ways